packages/create-angular/src/app/components/views/LanduseView.component.ts

```diff
@@ -0,0 +1,321 @@
+import {
+  Component,
+  computed,
+  ContentChild,
+  effect,
+  signal,
+} from '@angular/core';
+import { Color, Deck, MapViewState } from '@deck.gl/core';
+import { CardComponent } from '../Card.component';
+import { CardCollapsibleComponent } from '../CardCollapsible.component';
+import { FormulaWidgetComponent } from '../widgets/FormulaWidget.component';
+import { LayersComponent } from '../Layers.component';
+import { LegendEntryCategoricalComponent } from '../legends/LegendEntryCategorical.component';
+import { context } from '../../../context';
+import { AccessTokenService } from '../../services/AccessToken.service';
+import { Map } from 'maplibre-gl';
+import { BASEMAP, RasterTileLayer } from '@deck.gl/carto';
+import {
+  Filters,
+  getDataFilterExtensionProps,
+  RasterMetadata,
+  rasterSource,
+  vectorTilesetSource,
+} from '@carto/api-client';
+import { debouncedSignal } from '../../../utils';
+import { TreeWidgetComponent } from '../widgets/TreeWidget.component';
+import { DataFilterExtension } from '@deck.gl/extensions';
+
+const CONNECTION_NAME = 'amanzanares-pm-bq';
+const TILESET_NAME =
+  'cartodb-on-gcp-pm-team.amanzanares_raster.classification_us_compressed';
```

Comment:
Would `carto-demo-data.demo_rasters.usda_land_classification` from `carto_dw` be OK as a replacement here? @srtena